CPT331.WebAPI/packages.config

```diff
@@ -1,5 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Antlr" version="3.5.0.2" targetFramework="net452" />
+  <package id="Antlr3.Runtime.PCL" version="3.5.0.3" targetFramework="net452" />
+  <package id="bootstrap" version="3.3.7" targetFramework="net452" />
+  <package id="jQuery" version="1.9.1" targetFramework="net452" />
```

Comment:
Have these all been added by Swagger? If not, then let's only add what we're using.